### ISSUE TITLE
add 'host_name' varchar(255) for each 'server_id'

### ISSUE THREAD

**Issue by sreev** [zendesk/maxwell#593]
`

mysql> describe maxwell.heartbeats;
+-----------+------------------+------+-----+---------+-------+
| Field     | Type             | Null | Key | Default | Extra |
+-----------+------------------+------+-----+---------+-------+
| server_id | int(10) unsigned | NO   | PRI | NULL    |       |

`

**Comment by sreev** [comment]
i needed a hostname for each of these server_ids in maxwell database itself.
after doing a little more digging, it seems like the 'server_id' is a unique int, in mysql world.

i change my issue to add a new column 'host_name' varchar(255), so i have both 'server_id' and its 'host_name' available for business.

**Comment by sreev** [comment]
schemas, positions and heartbeats, these three tables has field 'server_id'.

it may be good idea to add a new table with 'server_hosts' with two fields 'server_id' and 'server_host_name'.

same for 'client_id' ? (may be in another ticket)

**Comment by sreev** [comment]
added entry in maxwell_schema.sql

> 
CREATE TABLE `server_hosts` (
  `server_id` int(10) unsigned NOT NULL,
  `server_host_name` varchar(255) NOT NULL,
  PRIMARY KEY (`server_id`)
);

updated MaxwellContext.java, added a method to insert values into this table.

**Comment by sreev** [comment]
development completed and testing in progress.

**Comment by osheroff** [comment]
why is the hostname needed?  not totally against it, just curious.

**Comment by sreev** [comment]
@osheroff 
we run, multiple maxwell processes per a sql host. so, it is convenient to see the sql host next to the server_id.

**Comment by sreev** [comment]
this functionality is in and tested with 1.8.0. requires rebase to the latest and & quick re-test.
but waiting on internal legal clearance to push out.

**Comment by osheroff** [comment]
closing this issue, feel free to reopen with more progress
